modules/cms/twig/DebugExtension.php

```diff
@@ -8,7 +8,7 @@
 use Cms\Classes\ComponentBase;
 use Illuminate\Pagination\Paginator;
 use Illuminate\Support\Collection;
-use Illuminate\Support\Debug\HtmlDumper;
```

Comment:
Does this require a class alias setup in the library?

Reply:
@LukeTowers good idea - https://github.com/octobercms/october/commit/0dee0e54b100a22cdd36647161e951af04effcce